wireguard-install.sh

```diff
@@ -121,28 +136,47 @@ function installWireGuard() {
 	# Install WireGuard tools and module
 	if [[ ${OS} == 'ubuntu' ]]; then
 		apt-get update
-		apt-get install -y wireguard iptables resolvconf qrencode
+		if [[ $Container == 1 ]]; then
+			apt-get install -y wireguard-tools --no-install-recommends
+			apt-get install -y iptables resolvconf qrencode
+		else
+			apt-get install -y wireguard iptables resolvconf qrencode
+		fi
+
 	elif [[ ${OS} == 'debian' ]]; then
 		if ! grep -rqs "^deb .* buster-backports" /etc/apt/; then
 			echo "deb http://deb.debian.org/debian buster-backports main" >/etc/apt/sources.list.d/backports.list
 			apt-get update
 		fi
 		apt update
 		apt-get install -y iptables resolvconf qrencode
-		apt-get install -y -t buster-backports wireguard
+		if [[ $Container == 1 ]]; then
+			apt-get install -y -t buster-backports wireguard-tools --no-install-recommends
+		else
+			apt-get install -y -t buster-backports wireguard
+		fi
+
 	elif [[ ${OS} == 'fedora' ]]; then
 		if [[ ${VERSION_ID} -lt 32 ]]; then
 			dnf install -y dnf-plugins-core
 			dnf copr enable -y jdoss/wireguard
-			dnf install -y wireguard-dkms
+			if [[ $Container != 1 ]]; then
+				dnf install -y wireguard-dkms
+			fi
 		fi
 		dnf install -y wireguard-tools iptables qrencode
+
 	elif [[ ${OS} == 'centos' ]]; then
 		yum -y install epel-release elrepo-release
 		if [[ ${VERSION_ID} -eq 7 ]]; then
 			yum -y install yum-plugin-elrepo
 		fi
-		yum -y install kmod-wireguard wireguard-tools iptables qrencode
+		if [[ $Container == 1 ]]; then
+			yum -y install wireguard-tools iptables qrencode
+		else
+			yum -y install kmod-wireguard wireguard-tools iptables qrencode
+		fi
```

Comment:
Install only wireguard-tools for Centos, as that is all that is needed when Kernel Module works.